src/CurrentBookRainbow/CurrentBookRainbow.js

```diff
@@ -55,47 +57,58 @@ const CurrentBookRainbow = ({ data, setClickedBook }) => {
         .on('click', (e, d) => {
           setClickedBook(d)
         })
+        .on("mouseover", function() {
+          d3.select(this).classed("hover", true);
+        })
+        .on("mouseout", function() {
+             d3.select(this).classed("hover", false);
+        })
```

Comment:
I'm so glad you added this! I think it will improve our UI so students know they can click on the book for more details!